src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -105312,7 +105438,7 @@ class Program
             comp.VerifyDiagnostics();
         }
 
-        // C<T~> operator op(C<T!>, C<T~>) where T : class
+        // C<T~> operator op(C<T!>, C<T~>) where T : class 2
```

Comment:
> 2 [](start = 58, length = 2)

Is this change intentional?

Reply:
Hah. No. Oh man am I ready for the weekend apparently. Thanks